.travis.yml

```diff
@@ -3,7 +3,7 @@ env:
     - RUST_BACKTRACE=1
     - PATH=$PATH:$HOME/.cargo/bin
 branches:
-  only: ci_test_run_refactor
+  only: fleming
```

Comment:
I think we don't need any restriction on the branch anymore. These scripts are only in this branch anyway.

Reply:
Right ok, no problem. Will remove this.